.rubocop.yml

```diff
@@ -21,3 +21,5 @@ Metrics/BlockLength:
   Exclude:
     - "**/*_spec.rb"
     - "db/schema.rb"
+Metrics/CyclomaticComplexity:
+  Max: 10
```

Comment:
Why do you set 10 ?

Reply:
case in ApplicationPolicy
@VasylYaremkiv

Reply:
You should fix rubocop warning and do not change rubocop config